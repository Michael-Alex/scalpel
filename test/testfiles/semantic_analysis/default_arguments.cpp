void f(int i = 0);
void g(int = 0);

