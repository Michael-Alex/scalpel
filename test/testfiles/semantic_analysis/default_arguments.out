{
  "id": "namespace-0",
  "simple functions":
  [
    {
      "id": "simple-function-0",
      "name": "f",
      "defined": false,
      "return type":
      {
        "fundamental type": "void"
      },
      "parameters":
      [
        {
          "name": "i",
          "type":
          {
            "fundamental type": "int"
          }
        }
      ]
    },
    {
      "id": "simple-function-1",
      "name": "g",
      "defined": false,
      "return type":
      {
        "fundamental type": "void"
      },
      "parameters":
      [
        {
          "type":
          {
            "fundamental type": "int"
          }
        }
      ]
    }
  ]
}
